components/edit-collective/sections/UserSecurity.js

```diff
@@ -193,17 +193,17 @@ class UserSecurity extends React.Component {
     }
   }
 
-  async disableTwoFactorAuth(values) {
+  async disableTwoFactorAuth() {
     try {
-      const { twoFactorAuthenticatorCode } = values;
       const account = {
         id: this.props.data.individual.id,
       };
 
+      this.setState({ disablingTwoFactorAuth: true, error: null });
```

Comment:
I guess you meant to use `isDisablingTwoFactorAuth` here. I guess we should rename these variables to avoid confusion.

Reply:
Done, thanks @kewitz !